lib/phoenix/socket.ex

```diff
@@ -1,10 +1,11 @@
 defmodule Phoenix.Socket do
-  @moduledoc """
+  @moduledoc ~S"""
```

Comment:
Makes you a superhero so you can fight batman in the next movie.

PS: it is the string sigil and ~S removes quoting / interpolation.